SpectraLogic.EscapePodClient/ResponseParsers/HeadResponseParser.cs

```diff
@@ -31,14 +34,23 @@ internal class HeadResponseParser : IResponseParser<bool>
 
         public bool Parse(IHttpWebResponse response)
         {
-            LOG.Debug(response.StatusCode);
-
-            if (response.StatusCode == HttpStatusCode.OK)
+            using (response)
             {
-                return true;
-            }
+                LOG.Debug(response.StatusCode);
+
+                if (response.StatusCode == HttpStatusCode.ServiceUnavailable)
+                {
+                    var errorResponse = JsonConvert.DeserializeObject<ErrorResponse>("{\"message\":\"The service is unavailable\",\"statusCode\":503}");
+                    throw new ErrorResponseException(errorResponse);
+                }
 
-            return false;
+                if (response.StatusCode == HttpStatusCode.OK)
```

Comment:
This seems like it can be simplified to `return response.statusCode == HttpStatusCode.OK;`